spec/factories/course.rb

```diff
@@ -36,6 +36,14 @@
       accredited_provider { create(:provider) }
     end
 
+    trait :primary do
```

Comment:
Is this to make it explicit? Because the level is already set to `primary`.

Reply:
Yes! :), most of the times I prefer explicit over implicitly.